spec/ProcessManager/UnitsRefundedProcessManagerSpec.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
Copyright/license block?

Reply:
We generally do not have them within the whole RefundPlugin 💃 could be added, but we can do it all at once in a separate PR